live/global.ini

```diff
@@ -22891,208 +22891,208 @@ PU_BANISTER_MG_Celebrate_IG_003_LoveIt=Ich liebe es!
-PU_BANISTER_MG_GenericChatter_IG_003_HeresSomethingYou=Hier ist etwas, das Sie vielleicht interessieren: Mein Nachname, Banister, rührt daher, dass meine Vorfahren vor etwa vierzig Generationen Meister im Treppenhandwerk waren. Alle wohlhabenden Leute konnten nicht genug von ihren Treppen bekommen. Mein Familienwappen sieht tatsächlich aus wie eines dieser "für den Notfall"-Schilder.
-PU_BANISTER_MG_GenericChatter_IG_004_LetMeBounce=Lass mich etwas von dir abprallen lassen. Ich überlege, ob ich für meinen Urlaub entweder nach Cassel oder nach Green fahre, und weißt du was, egal. Ich habe mich gerade entschieden. Tangaroa ist es.
-PU_BANISTER_MG_GenericChatter_IG_005_HeresAQuick=Hier ist ein kurzer Leckerbissen für Sie. Essen Sie niemals Toast, wenn die Schwerkraft ausgeschaltet ist. Ich habe noch nie in meinem Leben so viele Krümel gesehen. Riesiger Fehler.
-PU_BANISTER_MG_Greeting_1stX_IG_001_TessaToRandom=Tessa an zufälliges Schiff. Kommen Sie in einem zufälligen Schiff.
+PU_BANISTER_MG_FriendlyFire_IG_001_WatchYourAim=Achte auf Dein Ziel!
+PU_BANISTER_MG_FriendlyFire_IG_002_YoureNotSupposed=Darauf solltest du nicht schießen!
+PU_BANISTER_MG_FriendlyFire_IG_003_HoldYourFire=Feuer einstellen!
+PU_BANISTER_MG_GenericChatter_IG_001_YouKnowWhats=Weißt Du, was seltsam ist? Ich hatte neulich eine Fliege hier. Keine Ahnung, wie sie reinkam oder woher sie kam. Vielleicht war sie in einem meiner Lebensmittelpakete? Hat mich wahnsinnig gemacht, bis ich sie in eine Luftschleuse gejagt habe.
+PU_BANISTER_MG_GenericChatter_IG_002_IKeepRunning=Ich treffe immer wieder auf diese Leute, die Sternkartografie für langweilig halten. Kannst Du das glauben? Kartografie! Ich meine, wenn Sie der Meinung sind, dass es langweilig ist, Sternensysteme genau zu scannen und jedes einzelne astronomische Merkmal zu kartieren, dann weiß ich nicht einmal, was ich zu ihnen sagen soll.
+PU_BANISTER_MG_GenericChatter_IG_003_HeresSomethingYou=Hier etwas, das Du vielleicht interessant findest - mein Nachname, Banister, hat seinen Ursprung darin, dass meine Vorfahren vor etwa vierzig Generationen Meister im Treppenbau waren. Alle wohlhabenden Leute konnten nicht genug von ihren Treppen bekommen. Mein Familienwappen sieht tatsächlich aus wie eines dieser "Im Notfall"-Schilder.
+PU_BANISTER_MG_GenericChatter_IG_004_LetMeBounce=Lass mich etwas loswerden. Ich überlege, ob ich in meinem Urlaub nach Cassel oder nach Green fahren soll, und weißt du was, vergiss es. Ich habe mich gerade entschieden. Tangaroa ist es.
+PU_BANISTER_MG_GenericChatter_IG_005_HeresAQuick=Hier ist ein kleiner Tipp für Dich. Iss niemals Toast mit ausgeschalteter Schwerkraft. Ich habe in meinem Leben noch nie so viele Krümel gesehen. Großer Fehler.
+PU_BANISTER_MG_Greeting_1stX_IG_001_TessaToRandom=Tessa an unbekanntes Schiff. Kommen, unbekanntes Schiff.
```

Comment:
Dieser Satz ist sehr merkwürdig. Also auch in engl.: `Tessa to random ship. Come in random ship.`

Finden wir da einen eigenen Text?